src/controllers.js

```diff
@@ -1,35 +1,40 @@
-// import { re } from 'prettier';
-const request = require('request');
-
-const mainController = (req, res) =>
-  res.send({
-    message: 'Welcome to my jokes API!',
-  });
-
-const jokesController = (req, res) =>
-  request('https://api.icndb.com/jokes', (error, jokesApiResponse) => {
-    if (error) {
-      console.log(error);
-    }
-    const parsedResponse = JSON.parse(jokesApiResponse.body);
-
-    res.send({ jokes: parsedResponse.value });
-  });
-/* res.send({
-    message: 'This is the jokes endpoint',
-  }); */
-
-const randomJokesController = (req, res) =>
-  res.send({
-    message: 'No jokes here yet, try again!',
-  });
-const personalJokeController = (req, res) =>
-  res.json({
-    message: 'Hi, i can personalise your jokes!',
-  });
+const axios = require('axios');
+
+const jokesController = async (req, res) => {
+  try {
+    const response = await axios.get(`https://api.icndb.com/jokes`);
+
+    res.send({ jokes: response.data.value });
+  } catch (error) {
+    res.status(error.statusCode).send({ error: error.message });
+  }
+};
+
+const randomJokesController = async (req, res) => {
+  try {
+    const response = await axios.get(`https://api.icndb.com/jokes/random?exclude=[explicit]`);
```

Comment:
did you mean to hardcode removing explicit? Do we ask you to get rid of those? It would be a good extra learning and test to make endpoints for and without explicit param